core/local/steps/initial_diff.js

```diff
@@ -39,6 +38,9 @@ async function initialDiff (buffer, out, pouch) {
     const batch = []
     for (const event of events) {
       if (['created', 'modified', 'renamed', 'scan'].includes(event.action)) {
+        if (event.stats.fileid) {
+          byInode.delete(event.stats.fileid)
+        }
         byInode.delete(event.stats.ino)
```

Comment:
Why no conditions on the presence of `ino`? Do we have this property on Windows?

Reply:
We always have `ino`, even on windows with the new lib

Reply:
OK, I thought `fileId` was the concept of inode on Windows.

Reply:
It is, but nodejs uses `ino` for inode on linux & macOS, and for fileid on Windows (as a number). The issue is that we can have the same `ino` for two fileIds because the precision of number is not enough on JS. So, to improve things, we will use the `fileid` (as a string) from an external lib in the future, and will still use the `ino` to help the transition.